pkgs/development/python-modules/pycritty/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib, buildPythonPackage, fetchPypi, pythonOlder, pyyaml }:
+
+buildPythonPackage rec {
+  pname = "pycritty";
+  version = "0.3.5";
+
+  disabled = pythonOlder "3.6";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1lrmd4a1ps3h9z0pndfjfrd2qa7v3abd6np75fd2q2ffsqv7ar6x";
+  };
+
+  # The package does not include any tests to run
+  doCheck = false;
+
+  pythonImportsCheck = [ "pycritty" ];
+
+  propagatedBuildInputs = [ pyyaml ];
+
+  postPatch = ''
+    # remove custom install
+    substituteInPlace setup.py \
+      --replace "'install': PostInstallHook," ""
+  '';
```

Comment:
```suggestion
```

Reply:
@SuperSandro2000 If the custom `cmdclass` for this particular package is permitted to run, it attempts to create directories in your `$HOME` dir; that's the reason I disabled it. The package does not *need* these to run, and in fact fails to build on nix due to `$HOME` being set to the read-only `/homeless-shelter`.

What would be the most acceptable solution, here?

Reply:
Oh, I misunderstood the patch suggestion I think - this was a duplicate. My bad! Reading the Github interface is difficult at times 😓